### ISSUE TITLE
bug: the Revolt logo is unchanged in Light Mode, making it invisible

### ISSUE THREAD

**Issue by TGRush** [revoltchat/revite#676]
### What happened?

I expected the Revolt SVG to be inverted, turning it black, so that it would be read-able in Light Mode

This happens on all browsers, as well as the official Revite desktop client

### Branch

Production (app.revolt.chat)

### Commit hash

12b97160433151c188b077bd3e85fc2ee35bbd99

### What browsers are you seeing the problem on?

Other (please specify in the "What happened" form)

### Desktop

- [ ] Yes, this bug is specific to Revolt Desktop and is *not* an issue with Revolt Desktop itself.

### PWA

- [ ] Yes, this bug is specific to the PWA.

**Comment by Kapeet** [comment]
can you specify the places where this happens?

**Comment by TGRush** [comment]
So far I've only noticed this on the home page, though this mighr affect other areas where `wide.svg` is used

**Comment by insertish** [comment]
Closing due to rewrite, marking as potential issue in future by linking to https://github.com/revoltchat/frontend/issues/134.
